capellambse_context_diagrams/serializers.py

```diff
@@ -236,6 +236,7 @@ def get_styleclass(obj: common.GenericElement) -> str:
             (
                 styleclass[: -len("Component")],
                 "Human" * obj.is_human,
+                obj.nature.name.capitalize() if hasattr(obj, "nature") else "",
```

Comment:
No need to `capitalize()` here, because Enum member names are always in CAPS\_WITH\_UNDERSCORES anyways.

Reply:
Yes but I need a camelcased string here, i.e. `Node` or `Behavior`.

Reply:
Sorry, I got confused about `capitalize` and `upper`.